pkg/kubelet/cri/remote/remote_image.go

```diff
@@ -41,11 +41,12 @@ import (
 type remoteImageService struct {
 	timeout     time.Duration
 	imageClient runtimeapi.ImageServiceClient
+	logger      *klog.Logger
 }
 
 // NewRemoteImageService creates a new internalapi.ImageManagerService.
-func NewRemoteImageService(endpoint string, connectionTimeout time.Duration, tp trace.TracerProvider) (internalapi.ImageManagerService, error) {
-	klog.V(3).InfoS("Connecting to image service", "endpoint", endpoint)
+func NewRemoteImageService(endpoint string, connectionTimeout time.Duration, tp trace.TracerProvider, logger *klog.Logger) (internalapi.ImageManagerService, error) {
```

Comment:
should there be a new endpoint to not break end users? Or are we not worried about external callers to this yet?

Reply:
Right now it's not a real API, just when we move it to staging.